docs/CODE_GUIDELINES.md

```diff
@@ -366,43 +366,45 @@ static void test();
 **[back to top](#table-of-contents)**
 
 ## 7. Headers
-Included header files have to be sorted alphabetically to prevent duplicates and allow better overview, with an empty line clearly separating sections.
+Included header files have to be sorted (case sensitive) alphabetically to prevent duplicates and allow better overview, with an empty line clearly separating sections.
 
 Header order has to be:
 * Own header file
-* Other Kodi includes
+* Other Kodi includes (platform independent)
+* Other Kodi includes (platform specific)
 * C and C++ system files
```

Comment:
C++ before C?

Reply:
no in the same block

Reply:
why?

Reply:
it was like that before, if wanted it can be spilt into separate blocks

Reply:
No hard feelings. Just wanted to know whether there is other reason than "was like that before".

Reply:
I've been doing c++ before c but I'm indifferent